css/types/image.json

```diff
@@ -356,11 +356,28 @@
                 "edge": {
                   "version_added": "79"
                 },
-                "firefox": {
-                  "version_added": false
-                },
+                "firefox": [
+                  {
+                    "version_added": "83"
+                  },
+                  {
+                    "version_added": "75",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.conic-gradient.enabled"
+                      }
+                    ]
+                  }
+                ],
                 "firefox_android": {
-                  "version_added": false
+                  "version_added": "79",
```

Comment:
Reading https://bugzilla.mozilla.org/show_bug.cgi?id=1632351, it looks like this will also be in Firefox for Android 83. Can you copy the entry from desktop here too?

Reply:
Good catch, added. (Will mention this PR in that bug, just in case.)